pkg/subctl/cmd/show.go

```diff
@@ -98,3 +104,13 @@ func getClientConfigAndClusterName(rules *clientcmd.ClientConfigLoadingRules, ov
 
 	return restConfig{config: clientConfig, clusterName: *clusterName}, nil
 }
+
+func SubmarinerIsInstalled(err error) bool {
```

Comment:
```suggestion
func submarinerMissing(err error) bool {
```

As suggested, this name makes more sense as you're checking the error type that you're sending to the function.
If you'd have an independent function checking somehow that submariner is or isn't installed (not receiving an `error` variable) then the name you use would make sense.

Reply:
Done